.claude/hooks/web-research-gate.sh

```
@@ -1,65 +1,72 @@
+  FILE_PATH=""
 fi
 
 [ -z "$CONTENT" ] && exit 0
 
-# ── Pattern 1: SDK package imports ──
-HAS_SDK_IMPORT=""
-if echo "$CONTENT" | grep -qE '@anthropic-ai|@openai/codex|@google/gemini|claude-code|codex-sdk'; then
-  HAS_SDK_IMPORT="1"
+# Skip hook scripts themselves to avoid self-triggering on regex descriptions
+case "$FILE_PATH" in
+  */.claude/hooks/*|.claude/hooks/*|./.claude/hooks/*) exit 0 ;;
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 24.

(SC2221)